engines/dfc_provider/lib/dfc_provider.rb

```diff
@@ -11,6 +11,9 @@
 require "dfc_provider/address"
 require "dfc_provider/coordination"
 
+# 🙈
+require "inspect"
```

Comment:
I think that we should follow the convention to name the file after the class: `dfc_connector_inspect.rb`.